install/kots/manifests/gitpod-installer-job.yaml

```diff
@@ -191,7 +191,6 @@ spec:
                 --reset-values \
                 --timeout 1h \
                 --wait \
-                --wait-for-jobs \
```

Comment:
nit: Just wondering if this would falsely give a "Installer job finished" message. Is increasing the `ttl` of the job a possibility?

Reply:
I don't think so. The jobs are decoupled from the install/upgrade process really. Even if it does fail, it doesn't allow us to feedback to the GUI (I don't think KOTS has anything that allows that in fact)